src/main/java/com/finance/app/service/implementation/FinanceBaseServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.finance.app.service.implementation;
+
+
+import com.finance.app.infrastructute.exception.ContentNotFoundException;
+import com.finance.app.service.FinanceBaseService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+public class FinanceBaseServiceImpl<T> implements FinanceBaseService<T> {
```

Comment:
This class contain typical CRUD functionality... so the previous comment has more sense for now